### ISSUE TITLE
Battery module

### ISSUE THREAD

**Issue by meeDamian** [lncm/ideas#32]
This one should probably belong to a "case" repo, but we can move it there after @BTCln creates it.

@hitchcott has suggested [this] as our UPS solution. I think it might be perfect, but I also think we should try to find it cheaper somewhere.

<img width="1106" alt="macos 2018-11-07 at 14 27 52" src="https://user-images.githubusercontent.com/526657/48116657-56dd0880-e299-11e8-8341-0e691b79b65b.png">

(screenshot in case Amazon listing gets deleted/changed)

[this]: https://www.amazon.com/Raspberry-Battery-Charging-Protection-Output/dp/B07DB78PGJ/ref=pd_sbs_23_1?_encoding=UTF8&pd_rd_i=B07DB78PGJ&pd_rd_r=42980e83-e25c-11e8-9ca2-d5e819b39bab&pd_rd_w=wfWQc&pd_rd_wg=tW7Ve&pf_rd_i=desktop-dp-sims&pf_rd_m=ATVPDKIKX0DER&pf_rd_p=7d5d9c3c-5e01-44ac-97fd-261afd40b865&pf_rd_r=KCN78K18E09XHSCVG3F6&pf_rd_s=desktop-dp-sims&pf_rd_t=40701&psc=1&refRID=KCN78K18E09XHSCVG3F6

**Comment by nolim1t** [comment]
That looks cool. Maybe @NoahO might be able to find something when he goes to Shenzhen.

**Comment by meeDamian** [comment]
Yeah, I strongly believe that if we find it for a good price, it can be a really great and useful addition. 

Paging @BTCln as including it in a box design as early as possible might be a good idea - ideally in a way that's optional.
